### PR TITLE
feat: product page

### PR DESCRIPTION
**Pull request by vinusuhas-codecraft**
image

### CODE REVIEW

**Review by FayazCodeCraft — CHANGES_REQUESTED** [review]
Change border color to match the product

File: product/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
...iEIQCEIQElU5QPidN88p9jS1yqcoPxOm+eU+xoS8MPqPRHmnScqvJEeDI5H5igxuYZgdA0cDOQMcDA7AxwacQY8GEROMPxdXpWe1Z4iLPc4aGa6vSs9qzy0rz551wx3Ezy14LWskLOaiPzPRJp5rdnxI3MJWNFJiGJmGYOCRMxYkLyIRImYVtPJH8Rd6Se9ZNBme8kfxF3pJ71k0KePLmvoY/rCwhCRoQhCAQhCAkqfKF8Rp/ndPsaWyVPlD+I0/zyn2NCXhhtR6I806AzjUeiPNOgMjkeDFBjQYsB2YZiCLAcDHAznHAwG8KDNdXns95ZBrWTtf8AF1f+T3kkRRPX0Z+Lz9a+SuoPn75xORuM75iMAeudbHCXThmGYMpHm743My34OzFjMxMwmodmJmJmEAhCEDaeSL83u9JPesmhTPeSH83u9JPesmhTx5c19HHiFhCEjQhCEAhCEAlS5SNNY+gZ61Ltp7aryq9ZRG6ePMpJ+iW2NYAjB3g9h7YHy7U43jPUTjzTsJsfCvJjobrDYhejaOWRNk157wDvHmzjuAkcclelH/Wf9xJGO2sljhNa+C3Tfrn+rrh8Fum/XP8AV1wnbWTRRNY+C7Tfr3+rrifBdpv17/VpB21lMBNW+C7Tfr3+rSKOS7Tfr3+rrg7ax/hF3ChgCwQlmx8lgA3qIHrkWvWVkeUB4Hcftm1fBhp+y+weIRQfWDI68k+lHXYr+NmmXa/yMo+ydMM7j4Zy6Uy5ZB+E1/LX94Q/Ca/lr6xNi+CjSd9X8O/3kPgo0nfV/Dv95N/Pl6Y/zY+2OnU1/LX1icXtTsdfNtCbR8FGk76v4d/vInwUaTvq+of7yL1rfpZ/z4z7Ytz6fKH7whz6fKH7wm1fBRpO+r+Hf7yHwUaTvq/h3+8k+W+l+DH2xXn0+UP3hDn0+UP3hNq+CjSd9X8O/wB5E+CjSd9X1D/eR8t9HwY+2Lc+nyh6xGPqlG5TtMdwC78mbb8E+k76v4d/vJ24P5L9HXYHc7arvFdaClWP7TZL48Awz2yXq1Z0cYdyQ6GyvRM9ilTZZhc/pIo3t++zj/DL9OVVaooVVCqoCqqgBVAGAAB1CdZzdhCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIBCEIH//2Q==" alt="Product Image">
```
Comment (nikhilsnayak) [inline product/index.html:14]: This image URL looks ugly on markup. Can you use a better image URL?

**Review by nikhilsnayak — CHANGES_REQUESTED** [review]
- Can you reduce the width of your content card and place the image inside the card itself. I think it looks better

File: product/index.html (hunk 1 above)
Comment (vinusuhas-codecraft) [reply product/index.html]: the image url is shorter since its stored locally

**Comment by vinusuhas-codecraft** [comment]
> @FayazCodeCraft

i think i did some visual changes please have a look

**Review by nikhilsnayak — APPROVED** [review]
I think we can proceed with for now. Please stash all your commits into one meaning full commit for merging

**Review by FayazCodeCraft — APPROVED** [review]
LGTM

**Pull request merged**